src/agent/rustjail/src/container.rs

```diff
@@ -1002,38 +1002,24 @@ impl BaseContainer for LinuxContainer {
                 // Copy from stdin to term_master
                 if let Some(mut stdin_stream) = proc_io.stdin.take() {
                     let mut term_master = unsafe { File::from_raw_fd(pseudo.master) };
-                    let mut close_stdin_rx = proc_io.close_stdin_rx.clone();
-                    let wgw_input = proc_io.wg_input.worker();
                     let logger = logger.clone();
                     let term_closer = term_closer.clone();
                     tokio::spawn(async move {
                         let mut buf = [0u8; 8192];
                         loop {
-                            tokio::select! {
-                                // Make sure stdin_stream is drained before exiting
-                                biased;
-                                res = stdin_stream.read(&mut buf) => {
-                                    match res {
-                                        Err(_) | Ok(0) => {
-                                            debug!(logger, "copy from stdin to term_master end: {:?}", res);
-                                            break;
-                                        }
-                                        Ok(n) => {
-                                            if term_master.write_all(&buf[..n]).await.is_err() {
-                                                break;
-                                            }
-                                        }
-                                    }
+                            let res = stdin_stream.read(&mut buf).await;
```

Comment:
Here can we  use the tokio io copy directly?

Reply:
updated, thanks